### ISSUE TITLE
samba-server - 3.5.10-4: Fails to start, overwrites configuration

### ISSUE THREAD

**Issue by siavashs** [OpenBricks/openbricks#45]
I recently updated samba-server to 3.5.10-4
The service fails to start since `/run/geexbox/network/smbd` does not exist :
```
root@geexbox:~$ systemctl status smbd
smbd.service - Samba File Sharing Service
   Loaded: loaded (/usr/lib/systemd/system/smbd.service; disabled)
   Active: inactive (dead) since Wed 2015-04-15 22:33:20 IRDT; 30s ago
           start condition failed at Wed 2015-04-15 22:33:46 IRDT; 5s ago
           ConditionPathExists=/run/geexbox/network/smbd was not met
  Process: 23536 ExecStart=/usr/sbin/smbd -F (code=killed, signal=TERM)
 Main PID: 23536 (code=killed, signal=TERM)
```
Also the configuration file `/etc/smbd.conf` was overwritten without warning!

**Comment by tomlohave** [comment]
Le 15/04/2015 20:07, Siavash Safi a écrit :
> I recently updated samba-server to 3.5.10-4
> The service fails to start since |/run/geexbox/network/smbd| does not
> exist :
>
> |root@geexbox:~$ systemctl status smbd
> smbd.service - Samba File Sharing Service
>     Loaded: loaded (/usr/lib/systemd/system/smbd.service; disabled)
>     Active: inactive (dead) since Wed 2015-04-15 22:33:20 IRDT; 30s ago
>             start condition failed at Wed 2015-04-15 22:33:46 IRDT; 5s ago
>             ConditionPathExists=/run/geexbox/network/smbd was not met
>    Process: 23536 ExecStart=/usr/sbin/smbd -F (code=killed, signal=TERM)
>   Main PID: 23536 (code=killed, signal=TERM)
> |
Now, you need to explicitly set it to true in /etc/network or using the 
add-on program on xbmc
>
> Also the configuration file |/etc/smbd.conf| was overwritten without
> warning!
Ok, that's something we need to work on.

**Comment by warped-rudi** [comment]
The fact that smbd did not honor etc/network was a long standing bug. As for overwriting smb.conf: This should be fixed in 13a3d47f9931752a219fedcdd440a8334786c7b7
